internal/blueprint/blueprint_test.go

```diff
@@ -89,3 +90,20 @@ func TestBumpVersion(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPackages(t *testing.T) {
+
+	bp := Blueprint{
+		Name:        "packages-test",
+		Description: "Testing GetPackages function",
+		Version:     "0.0.1",
+		Packages: []Package{
+			{Name: "tmux", Version: "1.2"}},
+		Modules: []Package{
+			{Name: "openssh-server", Version: "8.*"}},
+		Groups: []Group{
+			{Name: "anaconda-tools"}},
+	}
+	Received_packages := bp.GetPackages()
+    assert.ElementsMatch(t, []string{"tmux-1.2", "openssh-server-8.*", "@anaconda-tools"} , Received_packages)
```

Comment:
Looks like this needs a `go fmt`?

Reply:
Ah yes, I got it formatted in the other commit. I'll move it here, thanks.